spec/models/manageiq/providers/amazon/aws_refresher_spec_common.rb

```diff
@@ -860,4 +862,51 @@ def assert_specific_vm_in_other_region
   def assert_relationship_tree
     expect(@ems.descendants_arranged).to match_relationship_tree({})
   end
+
+  def assert_specific_snapshot
```

Comment:
These specs here are just used to demonstrate what I plan to test once everything works. The legacy parser passes both specs.